Store/StoreAnalyticsOrderTracker.php

```diff
@@ -66,6 +69,27 @@ public function getFacebookPixelCommands()
 	}
 
 	// }}}
+	// {{{ public function getBingUETCommands()
+
+	public function getBingUETCommands()
+	{
+		$command = array(
+			'ec' => 'conversion',
+			'ea' => 'purchase',
+			'gv' => $this->getOrderTotal(),
+		);
+
+		$event_label = $this->getBingUETEventLabel();
+		if ($event_label != '') {
+			$command['el'] = $event_label;
+		}
+
+		$commands[] = $command;
+
+		return $commands;
```

Comment:
These 3 lines can be simplified to `return array($command);`. I'm not crazy about using array access on a variable that hasn't been defined. On the other hand PHP doesn't complain so :neutral_face:.